draft-ietf-httpbis-http2bis.xml

```diff
@@ -2688,19 +2688,11 @@
     <section anchor="HTTPLayer">
       <name>HTTP Message Exchanges</name>
       <t>
-        HTTP/2 is intended to be as compatible as possible with current uses of HTTP. This means
-        that, from the application perspective, the features of the protocol are largely
-        unchanged. To achieve this, all request and response semantics are preserved, although the
-        syntax of conveying those semantics has changed.
-      </t>
-      <t>
-        Thus, the specification and requirements of HTTP/1.1 Semantics and Content <xref target="RFC7231"/>, Conditional Requests <xref target="RFC7232"/>, Range Requests <xref target="RFC7233"/>, Caching <xref target="RFC7234"/>, and Authentication <xref target="RFC7235"/> are applicable to HTTP/2. Selected portions of HTTP/1.1 Message Syntax
-        and Routing <xref target="RFC7230"/>, such as the HTTP and HTTPS URI schemes, are also
-        applicable in HTTP/2, but the expression of those semantics for this protocol are defined
-        in the sections below.
+        HTTP/2 defines a framing of the abstract HTTP message abstraction
```

Comment:
Editorial nit: "abstract HTTP message abstraction". Aren't abstractions tautologically abstract? I think I'd propose deleting the first instance of the word "abstract".

```suggestion
        HTTP/2 defines a framing of the HTTP message abstraction
```